packages/@ama-sdk/schematics/cli/update-spec-from-npm.cts

```diff
@@ -44,8 +44,7 @@ const DEFAULT_SPEC_EXPORT_PATH_IN_NPM_MODULE = 'openapi';
 const supportedExtensions = [SPEC_JSON_EXTENSION, SPEC_YAML_EXTENSION, SPEC_YML_EXTENSION];
 
 if (help) {
-  // eslint-disable-next-line no-console
-  console.log(`This script can be used to update your local spec file from a given locally installed npm package.
+  logger.log(`This script can be used to update your local spec file from a given locally installed npm package.
```

Comment:
This might be stupid but with this version the command `update-spec-from-npm --help --quiet` will log nothing

Reply:
I can fix it